server/controllers/demo.js

```diff
@@ -0,0 +1,55 @@
+require("dotenv").config();
```

Comment:
This actually isn't required if we've already declared that in the bin/www to app.js. I would remove this.